src/useValidation/useValidation.ts

```diff
@@ -22,25 +22,31 @@ export type ValidateMethodOptions = {
 
 export type ValidateMethod = (options?: ValidateMethodOptions) => Promise<boolean>
 
-export type ResetMethodParams = [
-  /**
-   * An optional callback that will be called within a pause/resume block.
-   * This allows you to reset validation state and then reset the value
-   * without triggering another validation on change.
-   * For example if a value is required, use `reset(() => valueRef.value = undefined)`.
-   */
-  resetCallback?: () => void
-]
-export type ResetMethod = (...params: ResetMethodParams) => void
-
 export type UseValidation = ToRefs<UseValidationState> & {
   validate: ValidateMethod,
-  reset: ResetMethod,
+  /**
+   * Reset the validation state.
+  *
+  * @param resetCallback An optional callback that will be called within a pause/resume block.
+  *   This allows you to reset validation state and then reset the value without triggering
+  *   another validation on change.
+  *
+  * @example
+  * ```ts
+  * const value = ref(0)
+  * const { reset } = useValidation(value, isGreaterThanZero)
+  * reset(() => value.value = 0)
+  * ```
+  */
+  // eslint-disable-next-line @typescript-eslint/method-signature-style -- JSDoc in IDE doesn't work nearly as well with a property
```

Comment:
I played with this a _bunch_ and this was the best I could get to. Adding docs to a separate `ResetMethod` type gets a little lost and there's some finickyness with maintaining the `@param` linking as things get extracted out into TS types - seems like vs code stops tracking it as a function at various points...